### PR TITLE
 Mention how to customize unsafe operation styles

### PR DESCRIPTION
**Pull request by danielzfranklin**
I was confused about how to do this, so I filed #8474. I apologize for initially filing this incorrectly and spamming people.

### CODE REVIEW

File: docs/user/manual.adoc [hunk 1]
```diff
@@ -589,6 +589,24 @@ For example, mutable bindings are underlined by default and you can override thi
...
+                "function.unsafe": "#ff6600"
+                "method.unsafe": "#ff6600"
+            }
+        }
+    },
+}
+----
+
+Make sure you include the brackets around the theme name. For example, use "[Ayu Dark]" to customize the theme Ayu Dark.
+
```
Comment (matklad) [inline docs/user/manual.adoc:609]: It feels like the above two bits of advice duplicate each other a bit. Let's merge that into one code sample and explain the difference between top-level `rules` and `[Theme Name]` syntax.

**Comment by matklad** [comment]
friendly ping @danielzfranklin, would you like to address the comment?

**Comment by danielzfranklin** [comment]
Thanks for the reminder @matklad . I've addressed your comment.

**Comment by matklad** [comment]
Thanks!

bors r+

**Pull request merged**